docs/recipes/README.md

```diff
@@ -19,7 +19,9 @@ function* watchInput() {
 By using this helper the `watchInput` won't start a new `handleInput` task for 500ms, but in the same time it will still be accepting the latest `INPUT_CHANGED` actions into its underlaying `buffer`, so it'll miss all `INPUT_CHANGED` actions happening in-between. This ensures that the Saga will take at most one `INPUT_CHANGED` action during each period of 500ms and still be able to process trailing action.
 
 ## Debouncing
+From redux-saga@v1 [debounce](../api/README.md#debouncems-pattern-saga-args) is built-in effect.
 
+For `redux-saga@0.x`:
```

Comment:
Should we mention v0.x here? 😕 

I think the target readers are who uses v1.0, because other parts of  documentation is reflecting the v1.0 version and does not mention v0.x at all.

Reply:
sound legit

Reply:
should I remove the whole description how `retry` is implemented under-the-hood? imho it's quite useful for understanding the saga principles on the examples

Reply:
> should I remove the whole description how retry is implemented under-the-hood? 

no, i think (as you) that its helpful for the readers